tests/manage/cluster/nodes/test_nodes_maintenance.py

```diff
@@ -1,15 +1,17 @@
 import logging
 import pytest
 
+from ocs_ci.framework import config
 from ocs_ci.ocs import constants, node
+from ocs_ci.utility import aws
 from ocs_ci.framework.testlib import tier1, tier2, ManageTest, bugzilla
 
-from tests import sanity_helpers
+from tests.sanity_helpers import Sanity
 
 logger = logging.getLogger(__name__)
 
 
-@pytest.fixture()
+@pytest.fixture(autouse=True)
```

Comment:
This will be auto used before each test function, as this fixture is in 'function' scope.
so in case the test function is skipped (deselected) due to a bz decorator, the fixture will not take place